web/beacon-app/src/features/projects/types/updateProjectOwnerFormService.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable prettier/prettier */
+import { t } from '@lingui/macro';
+import { useFormik } from 'formik';
+import { object, string } from 'yup';
+
+import type { Project } from '../types/Project';
+
+export type UpdateProjectOwnerFormDTO = {
+  // select options types here
+  current_owner: {
+    label: string;
+    value: string;
+  };
+  new_owner: {
+    label: string;
+    value: string;
+  };
+};
+
+export const FORM_INITIAL_VALUES = {
+  current_owner: {
+    label: '',
+    value: '',
+  },
+  new_owner: {
+    label: '',
+    value: '',
+  },
+} satisfies UpdateProjectOwnerFormDTO;
+
+export const FORM_VALIDATION_SCHEMA = object({
+  new_owner: string().required(t`Please select a new owner.`),
```

Comment:
When I tried to save the `Change Owner` modal without selecting a different owner, I didn't see this error message.  I've included a screenshot of what I saw. 

Also, if an org only has 1 member, should `Change Owner` not appear as an option? This could be something that gets added in a separate story.

<img width="1572" alt="Screenshot 2023-05-15 at 10 15 47 AM" src="https://github.com/rotationalio/ensign/assets/94616884/0e140022-022e-4e61-9420-8d21f98d09b9">